packages/client/Usage.md

```diff
@@ -38,8 +38,8 @@ To get a collection of records, you can use the `Query` object. It provides the
 
 - `getFirst()`: returns the first record in the query results.
 - `getPaginated()`: returns a page of records in the query results.
-- `getAll()`: returns all the records in the query results.
-- `getMany()`: returns an array of some records in the query results.
+- `getAll()`: returns all the records in the query results. **We recommend avoiding this method and using `getMany` or `getPaginated` instead, as large datasets can cause performance problems in your applications.**
```

Comment:
```suggestion
- `getAll()`: returns all the records in the query results by paginating all the query results. If the query is not filtered and the table is a large dataset, this operation can affect the performance.
```

We should not recommend avoiding this method. There're a lot of cases where the table data is controlled, you don't have pagination and it is okay to use `getAll`. Also if the user properly filters the results, it is the recommended way to get all the records that match the query.